api/src/main/java/org/openmrs/api/ObsService.java

```diff
@@ -497,5 +499,51 @@ public Integer getObservationCount(List<Person> whom, List<Encounter> encounters
 	 */
 	@Authorized(PrivilegeConstants.GET_OBS)
 	public Integer getObservationCount(List<ConceptName> conceptNames, boolean includeVoided);
-	
+
+	/**
+	 * Creates or updates the given {@link ObsReferenceRange} in the database
+	 *
+	 * @param obsReferenceRange the obsReferenceRange to save
+	 *                             
+	 * @since 2.7.0
+	 * 
+	 * <strong>Should</strong> create a new obs reference range
+	 * 	 * <strong>Should</strong> edit an existing obs reference range
+	 */
+	ObsReferenceRange saveObsReferenceRange(final ObsReferenceRange obsReferenceRange);
```

Comment:
What is the access level of these methods?

Reply:
Public, but we don't need to add the modifier because the methods are implicitly public.